src/test/java/org/assertj/core/api/BDDAssertions_then_Test.java

```diff
@@ -105,7 +105,7 @@ public void then_Iterable() {
   @Test
   public void then_Iterator() {
     Iterator<String> iterator = Arrays.asList("1").iterator();
-    then(iterator).contains("1");
+    then(iterator).toIterable().contains("1");
```

Comment:
use `hasNext` instead of `contains` (do not use `toIterable()` we want to check `Iterator`)